app/views/tag/show/_header.html.erb

```diff
@@ -1,22 +1,34 @@
 <% if @wiki %>
   <div id="wiki-content">
-    <% if @wiki.main_image %>
-      <div class="row">
-        <div class="col-md-6" style="margin-bottom:10px;">
-          <%= image_tag(@wiki.main_image.path, class: "img-responsive") %>
-        </div>
-        <div class="col-md-6">
-          <h1 style="color:#333;"><%= link_to @wiki.latest.title, "/wiki/#{params[:id]}" %></h1>
-          <%= @wiki.latest.render_body.match(/<p>(.+)<\/p>/).to_a.try(:first).to_s.html_safe %>
-          <% if @wiki.latest.render_body.split('</p>').length > 2 %><a class="btn btn-lg btn-default" href="/wiki/<%= params[:id] %>">Learn more on the wiki page</a><% end %>
-        </div>
-      </div>
-    <% else %>
-      <h1><%= @wiki.latest.title %></h1>
-      <p><%= @wiki.latest.render_body.html_safe.match(/<p>(.+)<\/p>/).to_a.try(:first).to_s.html_safe %></p>
-      <p><%= link_to("Learn more on the wiki page", "/wiki/#{params[:id]}") if @wiki.latest.render_body.html_safe.match(/<p>(.+)<\/p>/).to_a.length > 1 %></p>
-    <% end %>
+      <div class="banner-text-w3ls">
+            <div class="container" id="home">
+                <div class="card cardy">
+                    <div class="box">
+			 <div class="img">
+                            <%= image_tag(@wiki.main_image.path, class: "img-responsive" %>
+                         </div>
+
+		       <% if @wiki.main_image %>
+        		<h2 style="color:#333;"><%= link_to @wiki.latest.title, "/wiki/#{params[:id]}" %></h2>
+          		<%= @wiki.latest.render_body.match(/<p>(.+)<\/p>/).to_a.try(:first).to_s.html_safe %>
+          		<div id="outer">
+                            <div class="button_slide slide_left"><i class="fas fa-external-link-alt"></i> Follow Topic</a>
+                            </div>
+			</div>
+        	    </div>
+
+		       <% else %>
+      			<h2><%= @wiki.latest.title %></h2>
+      			<p><%= @wiki.latest.render_body.html_safe.match(/<p>(.+)<\/p>/).to_a.try(:first).to_s.html_safe %></p>
+      			<div id="outer">
+          		    <div class="button_slide slide_left"><i class="fas fa-external-link-alt"></i> Follow Topic</a></div>
+      			</div>
+    		       <% end %>
+      		</div>
+
+	    </div>
   </div>
+
 <% elsif @wildcard %>
```

Comment:
Aha - so you removed the conditional code above, so this `elsif` is left alone here! I think you may have to preserve the `if @wiki` conditionals to make this work!